catalog/transformer/catalog-transformer-bootflag/src/main/java/org/codice/ddf/catalog/transformer/bootflag/InputTransformerBootServiceFlag.java

```diff
@@ -51,7 +51,7 @@
       "org.codice.ddf.platform.bootflag.transformerWaitTimeoutSeconds";
 
   private static final long DEFAULT_TRANSFORMER_WAIT_TIMEOUT_SECONDS =
-      TimeUnit.MINUTES.toSeconds(5);
+      TimeUnit.MINUTES.toSeconds(20);
```

Comment:
❓ Do we need the timeout to be this large on a normal system, or is the issue only with itests in pax exam? Can we just increase the timeout for itests via the `org.codice.ddf.platform.bootflag.transformerWaitTimeoutSeconds` system property?

Reply:
We don't even need to increase the timeout for itests anymore. Itests taking a long time was resolved by the feature/bundle dependency fix. I'm thinking we should just leave the blueprint timeout at 30 minutes but set the default for `org.codice.ddf.platform.bootflag.transformerWaitTimeoutSeconds` back to 5 minutes. @peterhuffer @emmberk do you agree?